MaxText/configs/base.yml

```diff
@@ -61,6 +61,7 @@ save_config_to_gcs: False
 dtype: "bfloat16"
 quantization: "" #defaults to no quantization, i.e. bf16. possible alternative setting is 'int8' or use fp8 to run with 8-bit floating-point GeMMs on NVIDIA GPUs.
 quantize_kvcache: False
+kv_quant_axis: "kd"
```

Comment:
Can you add comment with possible values for this param?

Reply:
@singh-mitali, FYI

Reply:
Can we use more descriptive names here? I realize we don't have a great name for head dim - maybe dkv - so the three could be "dkv", "heads_and_dkv", "dkv_broadcast"

I don't see the differnce between "dkv" and "dkv_broadcast" in behavior/code though? What is meant by broadcast here?

Reply:
I had a hard time parsing the first sentence of this comment, I think what is meant is that kv_quant_axis is only applied when quantize_kvcache is True?

Reply:
Maybe also add a sentence or two explaining the tradeoff - correct/modify this if wrong:

More accurate quantiziation is achieved by using finer grained scale calibrations across more axes, however this comes with the cost of additional compute time.